controller/linodemachine_controller_helpers.go

```diff
@@ -63,21 +63,14 @@ var (
 	errNoPublicIPv6SLAACAddrs = errors.New("no public SLAAC address set")
 )
 
-func handleTooManyRequestsError(err error) (ctrl.Result, error) {
-	newErr := linodego.NewError(err)
-	if newErr.Response == nil {
-		return ctrl.Result{RequeueAfter: reconciler.DefaultLinodeTooManyRequestsErrorRetryDelay}, nil
-	}
-	if newErr.Response.Request.Method != http.MethodPost {
-		return ctrl.Result{RequeueAfter: reconciler.DefaultLinodeTooManyRequestsErrorRetryDelay}, nil
-	}
-	return ctrl.Result{RequeueAfter: reconciler.DefaultLinodeTooManyPOSTRequestsErrorRetryDelay}, nil
+func handleTooManyRequestsError() (ctrl.Result, error) {
```

Comment:
Since we're only using this in one place and it's one line, we should probably get rid of it

Reply:
Yeah, makes sense. Let me update it. The only drawback of not keeping the default limit of 15 secs in a var and using it to retry faster for POST calls is that if there are more than 1 clusters using token from same account, one can end up using all allowed rate-limit requests for the account and other will fail with 429 error. Since the call will fail, new code will simply retry after 1 min where it should have retried after 15 secs for POST calls. Though the possibility of this happening is low as its only for POST calls and it looks like it will be ok to wait for 1 min in such edge cases.

Reply:
Updated.